UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -346,16 +346,20 @@ public void RemoveInventoryAndBody(Vector3 AppearAtWorld)
 		/// </summary>
 		public void TryRemoveFromBody(bool beingGibbed = false, bool CausesBleed = true, bool Destroy = false, bool PreventGibb_Death = false) //TODO It should do the stuff automatically when removed from inventory
 		{
+			if ( HealthMaster == null ) return;
 			bool alreadyBleeding = false;
 			if (CausesBleed)
 			{
-				foreach (var bodyPart in HealthMaster.BodyPartList)
+				if (HealthMaster != null)
 				{
-					if (bodyPart.BodyPartType == BodyPartType.Chest && alreadyBleeding == false)
+					foreach (var bodyPart in HealthMaster.BodyPartList)
 					{
-						bodyPart.IsBleeding = true;
-						alreadyBleeding = true;
-						HealthMaster.ChangeBleedStacks(limbLossBleedingValue);
+						if (bodyPart.BodyPartType == BodyPartType.Chest && alreadyBleeding == false)
+						{
+							bodyPart.IsBleeding = true;
+							alreadyBleeding = true;
```

Comment:
As far as I can see, alreadyBleeding is only used to prevent this block from triggering for multiple body parts - in that case this variable can be removed as just be replaced with a break after HealthMaster.ChangeBleedStacks().

Reply:
idk, I just added some stuff so it won't error   if health Master is null